docs/modules/getting_started/examples/snip_crc_preset_platforms.adoc

```diff
@@ -0,0 +1,25 @@
+.Preset and architecture compatibility
+|===
+| Preset | AMD64 | Intel 64 | M1
+
+| {ocp}
```

Comment:
this makes it hard to read/track for me...
this is style or required?
would expect `| {preset} | yes | yes | no |` to be possible

Reply:
Right, and even better, CSV is a more common format.